### ISSUE TITLE
#Bug Icons are not being rendered

### ISSUE THREAD

**Issue by calvinomiguel** [atisawd/boxicons#252]
The expand vertical and expand horizontal icons aren't being rendered when used in the code `<i class='bx bx-expand-vertical' ></i>` `<i class='bx bx-expand-horizontal' ></i>`.

**Comment by atisawd** [comment]
Are you using the newest version of boxicons?

**Comment by calvinomiguel** [comment]
> Are you using the newest version of boxicons? They were added in 2.1.0

Yes am I. I just realized, at least on my project, the issue happens only on when I use this icon as a font `<i class="bx bx-expand-vertical"></i>` 

if I use it as a webcomponent  `<box-icon name="expand-vertical"></box-icon>` then it's being displayed without problems.
